modules/blockexplorer/blockexplorer.go

```diff
@@ -38,21 +58,28 @@ func New(cs modules.ConsensusSet) (bc *BlockExplorer, err error) {
 	}
 
 	// Initilize the module state
-	bc = &BlockExplorer{
+	be = &BlockExplorer{
 		currentBlock:     cs.GenesisBlock(),
 		blockchainHeight: 0,
+		currencySent:     types.NewCurrency64(0),
+		fileContracts:    0,
+		fileContractCost: types.NewCurrency64(0),
+		blocks:           make([]modules.ExplorerBlockData, 0),
```

Comment:
this also applies to slices, the default is nil, but append() can accept nil values